src/main/java/de/hpi/bpt/argos/api/entity/EntityEndpointImpl.java

```diff
@@ -0,0 +1,310 @@
+package de.hpi.bpt.argos.api.entity;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import de.hpi.bpt.argos.api.eventType.EventTypeEndpointImpl;
+import de.hpi.bpt.argos.storage.PersistenceAdapterImpl;
+import de.hpi.bpt.argos.storage.dataModel.attribute.Attribute;
+import de.hpi.bpt.argos.storage.dataModel.attribute.type.TypeAttribute;
+import de.hpi.bpt.argos.storage.dataModel.entity.Entity;
+import de.hpi.bpt.argos.storage.dataModel.event.Event;
+import de.hpi.bpt.argos.storage.dataModel.event.type.EventType;
+import de.hpi.bpt.argos.util.HttpStatusCodes;
+import de.hpi.bpt.argos.util.LoggerUtilImpl;
+import de.hpi.bpt.argos.util.RestEndpointUtil;
+import de.hpi.bpt.argos.util.RestEndpointUtilImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static spark.Spark.halt;
+
+/**
+ * {@inheritDoc}
+ * This is the implementation.
+ */
+public class EntityEndpointImpl implements EntityEndpoint {
+
+    private static final Logger logger = LoggerFactory.getLogger(EventTypeEndpointImpl.class);
+    private static final Gson serializer = new Gson();
+    private static final RestEndpointUtil endpointUtil = RestEndpointUtilImpl.getInstance();
+    private static final JsonParser jsonParser = new JsonParser();

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "jsonParser" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1068)